src/libstore/store-api.hh

```diff
@@ -164,6 +164,10 @@ public:
 
     Setting<bool> wantMassQuery{this, false, "want-mass-query", "whether this substituter can be queried efficiently for path validity"};
 
+    Setting<StringSet> systemFeatures{this, settings.systemFeatures,
```

Comment:
The old setting is still there for backwards compat. It just is used to default the store setting.